package.json

```diff
@@ -82,6 +92,28 @@
 				"title": "File Access: What's New"
 			}
 		],
+		"menus": {
+			"explorer/context": [
+				{
+					"command": "readOnly.makeReadOnlyRecursive",
+					"when": "isWindows && explorerResourceIsFolder"
```

Comment:
Instead of a _folder only_ command, it could work on both (files and folders) and leave the command itself decide how to do. If a folder, ask/warn about being recursive.